spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+           1. Let _iv_ be the substring of _s_ from position 0, inclusive, to position _dp_, exclusive.
+           1. Let _fv_ be the substring of _s_ from position _dp_, exclusive.
+           1. Let _f_ be ? ToNumber(_fv_).
+           1. Let _v_ be ? ToLength(? Get(_fv_, `"length"`)).
+         1. Let _i_ be ? abs(? ToNumber(_iv_)).
+         1. If _f_ ≠ 0, then
+           1. Let _ft_ be the value of _fv_ stripped of trailing `"0"`.
+           1. Let _w_ be ? ToLength(? Get(_ft_, `"length"`)).
+           1. Let _t_ be ? ToNumber(_ft_).
+         1. Else,
+           1. Let _w_ be 0.
+           1. Let _t_ be 0.
+         1. Return a new Record { [[Number]]: _n_, [[IntegerDigits]]: _i_, [[NumberOfFractionDigits]]: _v_, [[NumberOfFractionDigitsWithoutTrailing]]: _w_, [[FractionDigits]]: _f_, [[FractionDigitsWithoutTrailing]]: _t_ }.
+       </emu-alg>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Internal Slots of Plural Rules Operands</emu-caption>
+         <table class="real-table">
+           <thead>
+             <tr>
+               <th>Internal Slot</th>
+               <th>Type</th>
+               <th>Description</th>
+             </tr>
+           </thead>
+           <tr>
+             <td>[[Number]]</td>
+             <td>Number</td>
+             <td>Absolute value of the source number (integer and decimals)</td>
+           </tr>
+           <tr>
+             <td>[[IntegerDigits]]</td>
+             <td>Number</td>
+             <td>Number of digits of [[Number]].</td>
+           </tr>
+           <tr>
+             <td>[[NumberOfFractionDigits]]</td>
+             <td>Number</td>
+             <td>Number of visible fraction digits in [[Number]], <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[NumberOfFractionDigitsWithoutTrailing]]</td>
+             <td>Number</td>
+             <td>Number of visible fraction digits in [[Number]], <i>without</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[FractionDigits]]</td>
+             <td>Number</td>
+             <td>Number of visible fractional digits in [[Number]], <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[FractionDigitsWithoutTrailing]]</td>
+             <td>Number</td>
+             <td>Number of visible fractional digits in [[Number]], <i>without</i> trailing zeros. </td>
+           </tr>
+         </table>
+       </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-pluralruleselection" aoid="PluralRuleSelection">
+
+      <h1>PluralRuleSelection ( _locale_, _type_, _n_, _operands_ )</h1>
+
+      <p>
+        When the PluralRuleSelection abstract operation is called with four arguments, it performs implementation dependent algorithm to map _n_ to the appropriate plural representation of the Operand Record _operands_ by selecting the rules denoted by _type_ for the corresponding _locale_, or the String value `"other"`.
```

Comment:
- /performs implementation dependent/performs an implementation-dependent/  (missing indefinite article and hyphen)
- /Operand Record/Plural Rules Operands Record/ (maybe add `<emu-xref>` ?)